src/components/Counter.js

```diff
@@ -0,0 +1,35 @@
+import React,{useContext} from 'react';
+import Button from '@mui/material/Button';
+import { BoxCounter } from './boxCounter';
+import { CountContext } from './CountContext';
+export function Counter() {
+    const {add,setCount}=useContext(CountContext);
+
+    const updateCart=()=>{
+        setCount(add);
+    };
+
+    return (
+        <div className='Counter'>
+            <BoxCounter />
+            <div>
+                <Button
+                    style={{
+                        backgroundColor: 'hsl(26, 100%, 55%)',
```

Comment:
why dont re-use ButtonMaterail UI?

Reply:
The properties are very different, or should i create a new component?

Reply:
you can leave this implementation, but consider that if a component is created a unique, like Buttonmui, the idea is to use it around all aplication. That means that modifications or updates, should be handle by the props.